pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -174,8 +192,9 @@ public void init(SegmentGeneratorConfig segmentCreationSpec, SegmentIndexCreatio
           .withIndexDir(_indexDir)
           .withCardinality(columnIndexCreationInfo.getDistinctValueCount())
           .withDictionary(dictEnabledColumn)
-          .withFieldSpec(fieldSpec)
-          .withTotalDocs(segmentIndexCreationInfo.getTotalDocs())
```

Comment:
Can we keep these one per line? It’s a lot easier to read in my opinion, and is very standard with builders.